test/unit/controllers/declaration/DeclarationHolderControllerSpec.scala

```diff
@@ -201,5 +201,47 @@ class DeclarationHolderControllerSpec extends ControllerSpec with ErrorHandlerMo
         status(result) must be(OK)
       }
     }
+
+    "should redirect to Origination Country page" when {
+
+      "user is during Supplementary journey" in new SetUp {
```

Comment:
What do you think about remove setup trait

Reply:
Old spec, just added new case, want to be consistent with rest of the tests. Don't want to spend a lot of time now on refactoring. The ticket is huge, so I need to first finish it.